.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: htsspecs/circle-ci-image:0.4
+    steps:
+      - checkout
+      - run: 
+          name: Compile the LaTeX
+          command: |
+              make
+              mkdir pdfs
+              cp new/*.pdf pdfs
+              mkdir logs
+              cp new/*.log logs
+
+      - run: 
+          name: Compile the diff LaTeX files
+          command: |
+              mergebase=$(git merge-base HEAD origin/master)
+              echo $mergebase
+
+              mkdir -p diffs
+              make --touch diffs
+
+              # Avoid timing issues
+              sleep 2  
+
+              touch $(git diff-tree --name-only $mergebase HEAD) || true
+
+              # Rebuild the actually changed PDFs, and remove the merely touched ones
+              make diffs
+              find diffs -empty -delete -mindepth 1
```

Comment:
Log file messages show that `-mindepth` is used incorrectly here.

Reply:
fixed

Reply:
Not fixed. It still always logs a warning:

> find: warning: you have specified the -mindepth option after a non-option argument -empty, but options are not positional (-mindepth affects tests specified before it as well as those specified after it).  Please specify options before other arguments.